packages/node_modules/@ciscospark/internal-plugin-mercury/src/mercury.js

```diff
@@ -51,7 +51,7 @@ const Mercury = SparkPlugin.extend({
   },
 
   @oneFlight
-  connect() {
+  connect(webSocketUrl) {
     if (this.connected) {
```

Comment:
Let's get some unit tests that confirm the underlying socket is created with webSocketUrl when specified or the default when not (I assume one of those tests probably already exists, but it seems like this change should have needed at least one new test.)